src/main/groovy/org/mastercoin/test/TestSupport.groovy

```diff
@@ -265,7 +265,7 @@ trait TestSupport implements MastercoinClientDelegate {
      * @param amount     The number of units to create
      * @return The transaction hash
      */
-    Sha256Hash createProperty(Address address, Ecosystem ecosystem, PropertyType type, Long amount) {
+    Sha256Hash createProperty(Address address, Ecosystem ecosystem, PropertyType type, Number amount) {
```

Comment:
At the moment, there is no specific `NumberOfCoins` class like `CurrencyID` and the numbers get converted into hex via `longValue()`, which is a part of the interface `Number`, so it seemed intuitive to use the interface directly instead of `Long`, which allows to use `createProperty` also with `BigDecimal` while maintaining the correct conversion.